src/main/java/concatenatedwordschallenge/util/TaskFileReader.java

```diff
@@ -0,0 +1,46 @@
+package concatenatedwordschallenge.util;
+
+import org.apache.log4j.Logger;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+
+public class TaskFileReader {
+
+    private static Logger logger = Logger.getLogger(TaskFileReader.class);
+
+    private List<String> words;
+
+    public TaskFileReader (String fileName) {
+        this.words = getWordsFromFile(getPathToResourcesFile(fileName));
+    }
+
+    private List<String> getWordsFromFile(String path) {
+        String wordsFromFile = "";
+        try {
+            wordsFromFile = Files.readString(Paths.get(path));
+        } catch (IOException e) {
+            logger.error(e);
```

Comment:
```suggestion
            logger.error("Can't do something with something: " + something, e);
```
always add your custom text to error message. it makes easier to handle